### ISSUE TITLE
NVDA doesn't read Microsoft Edges configuration menu with mouse

### ISSUE THREAD

**Issue by Adriorjalesvidal** [nvaccess/nvda#10061]
### Steps to reproduce:
1. Open Microsoft Edge
2. Move the mouse to find Microsoft Edge's more options menu
3. Open the menu and use the mouse to navigate it, you may realise that NVDA doesn't read it.

### Actual behavior:
NVDA doesn't read the menu items of the Microsoft Edge's configurations and options menu using mouse

### Expected behavior:
NVDA must read it using the mouse

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
NVDA 2019.1.1

#### Windows version:
Windows 10 1903, may 2019 update

#### Name and version of other software in use when reproducing the issue:
Microsoft Edge

#### Other information about your system:
Computer: Dell Vostro 15-5568
Memory: 16 GB
Processor: Intel core TM I5 Caby lake, (7th generation)
Disk: 2 ssd disks, 256+1024 GB
Computer up to date, Windows 10 1903-Professional

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

**Comment by Adriani90** [comment]
This works as expected in Edge Cromium 81 with NVDA Version: alpha-20015,56a62097. 
Closing as works for me, since Edge Cromium will be distributed to most users in Windows 20 h1 update.

**Comment by Adriani90** [comment]
I think this is only reproducible in the ol Edge Spartan. Addin appropriate label.
